weblate/screenshots/models.py

```diff
@@ -76,3 +83,24 @@ def change_screenshot_assignment(sender, instance, action, **kwargs):
         name="UnusedScreenshot"
     ).exists():
         component_alerts.delay([instance.pk])
+
+
+def update_screenshots_from_repo(self, component):
+    screenshots = Screenshot.objects.filter(translation__component=component).order_by(
+        "id"
+    )
```

Comment:
Ordering is not really needed here.

```suggestion
    screenshots = Screenshot.objects.filter(translation__component=component)
```